src/app/features/auth/components/login-form/login-form.component.html

```diff
@@ -11,7 +11,11 @@ <h3>Sign In</h3>
       placeholder="Email"
       type="email"
     />
-    <validation-message class="text-red-500 text-sm mt-1" [control]="form.controls['email']" />
+    <validation-message
+      *ngIf="form.controls['email'].touched"
+      class="text-red-500 text-sm mt-1"
```

Comment:
These classes repeats everywhere. Maybe add them to the component host internally?